elpis/engines/espnet/objects/model.py

```diff
@@ -107,7 +111,7 @@ def train():
             print(f"SELF PATH {self.path}")
             if os.path.isfile(run_log_path):
                 os.remove(run_log_path)
-            p = run(f"cd {local_espnet_path}; ./run.sh --nj 1 &> {run_log_path}")
+            p = run(f"cd {local_espnet_path}; ./run.sh --ngpu {self.config['gpus']} --nj 1 &> {run_log_path}")
```

Comment:
Does this work for ngpu=0?

Reply:
If I recall well, yes, because ngpu=0 means cpu mode.